### PR TITLE
update component index, alphabetize some out-of-order items

### PR DESCRIPTION
**Pull request by davidcalhoun**
#### What does this PR do?
Updates the components index, which exposes missing components on the `Grommet` namespace for projects that use the minified JS (such as the Codepen examples).

Also fixes the ordering of a few components that weren't alphabetized correctly.

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -25,19 +26,22 @@ var Grommet = {
     Range: require('./components/chart/Range')
   },
   CheckBox: require('./components/CheckBox'),
+  Collapsible: require('./components/Collapsible'),
   Columns: require('./components/Columns'),
   DateTime: require('./components/DateTime'),
+  DateTimeDrop: require('./components/DateTimeDrop'),
```
Comment (alansouzati) [inline src/js/index.js]: I believe this is an internal component and should not be exposed.

File: src/js/index.js [hunk 2]
```diff
@@ -25,19 +26,22 @@ var Grommet = {
     Range: require('./components/chart/Range')
   },
   CheckBox: require('./components/CheckBox'),
+  Collapsible: require('./components/Collapsible'),
```
Comment (alansouzati) [inline src/js/index.js]: I believe this is an internal component and should not be exposed.

File: src/js/index.js [hunk 3]
```diff
@@ -1,14 +1,15 @@
...
-  Animate: require('./components/Animate'),
   Accordion: require('./components/Accordion'),
   AccordionPanel: require('./components/AccordionPanel'),
   Anchor: require('./components/Anchor'),
+  Animate: require('./components/Animate'),
   App: require('./components/App'),
   Article: require('./components/Article'),
   Box: require('./components/Box'),
   Button: require('./components/Button'),
+  Card: require('./components/Card'),
```
Comment (alansouzati) [inline src/js/index.js]: Thanks for adding Card!

**Comment by alansouzati** [comment]
Thanks that looks good, there are two components that I believe should not be exposed.

File: src/js/index.js (hunk 1 above)
Comment (davidcalhoun) [inline src/js/index.js]: Sounds good - I remove it and added a comment

File: src/js/index.js (hunk 2 above)
Comment (davidcalhoun) [inline src/js/index.js]: Removed it and added a note

**Comment by davidcalhoun** [comment]
@alansouzati Removed Collapsible and DateTimeDrop.  Should be good to go now.  Thx for having a look!

A squash and merge can be performed from the UI dropdown below :)

**Comment by alansouzati** [comment]
beautiful did not know about the squash feature, thanks!!!!

**Pull request merged**